src/get-element/getElement.js

```diff
@@ -9,31 +9,25 @@
 const getElement = (cssSelectors, outTimer = 10000, onError = null) => {
   let selectors = Array.isArray(cssSelectors) ? cssSelectors : [cssSelectors];
   let results = {};
-  let intervalId;
 
-  const clearTimer = () => {
-    if (intervalId) {
-      clearInterval(intervalId);
-    }
-  };
-
-  const checkElements = () => {
-    selectors = selectors.filter((selector) => {
-      const els = document.querySelectorAll(selector);
-      if (els.length > 0) {
-        results[selector] = els;
-        return false; // Remove from selectors to check
+  return new Promise((resolve, reject) => {
+    const observer = new MutationObserver(() => {
+      selectors = selectors.filter((selector) => {
```

Comment:
Do we have the goal or requirement to avoid the use of jQuery or sizzle?
jQuery could greatly simplify this block, jQuery selectors, just like CSS, allow more than 1 selector separated by a comma. just doing `jQuery(selectors.join(', ')).length>0` can tell you if any of the selectors return an item.

...just a thought

Reply:
I like this idea, I think I'm going to wait to add it if the logic gets more complicated. I think I want to keep fe-dev-utils pretty small for now and avoid bundling any dependencies with it for now.